packages/odyssey-react-mui/src/labs/SideNav/NavAccordion.tsx

```diff
@@ -103,6 +100,7 @@ const AccordionSummaryContainer = styled(MuiAccordionSummary, {
 
   ...(isCompact && {
     paddingBlock: odysseyDesignTokens.Spacing2,
+    minHeight: odysseyDesignTokens.Spacing4,
```

Comment:
We can remove this min-height

Reply:
as above, this sets the list items to a consistent height

Reply:
I'd recommend setting `height` to that value then.

Reply:
Do we intend for these to wrap? If so, we should be using `min-height`. But if we don't want wrapping, then `height`.

Reply:
Yeah, I would assume the text should wrap if you have a longer Title. that's why the Mui Accordion has the min-height set instead.

Reply:
The text will wrap regardless of the items height/min-height. If we set an explicit height the text will overflow out of the container. To remove the default Accordion min-height, I'd go with `minHeight: "unset"` here then